opencodeblocks/graphics/view.py

```diff
@@ -149,16 +149,23 @@ def centerView(self, x: float, y: float):
     def moveToGlobalView(self) -> bool:
```

Comment:
Maybe the name is misleading now ?
Should be something like moveToItems maybe ?